OctopusDSC/DSCResources/cOctopusServer/cOctopusServer.psm1

```diff
@@ -1455,7 +1465,8 @@ function Test-ParameterSet {
         [string]$DownloadUrl,
         [string]$WebListenPrefix,
         [string]$SqlDbConnectionString,
-        [PSCredential]$OctopusAdminCredential = [PSCredential]::Empty
+        [PSCredential]$OctopusAdminCredential = [PSCredential]::Empty,
+        [PSCredential]$OctopusMasterKey = [PSCredential]::Empty
```

Comment:
These default values aren't really needed, as this is an internal function (I know that its not already consistently applied)